### PR TITLE
Add tests for reading account data

### PR DESCRIPTION
**Pull request by turt2live**
See https://github.com/matrix-org/synapse/pull/4303

This was done with a community hat on:
Signed-off-by: Travis Ralston <travis@t2bot.io>

### CODE REVIEW

File: tests/10apidoc/44account_data.pl [hunk 1]
```diff
@@ -41,6 +42,42 @@ sub matrix_add_room_account_data
    );
 }
 
+=head2 matrix_get_account_data
+
+   matrix_get_account_data( $user, $type, $content )->get;
```
Comment (erikjohnston) [inline tests/10apidoc/44account_data.pl]: why does this have a `->get`?

Comment (turt2live) [reply tests/10apidoc/44account_data.pl]: copy/paste. I guess it's not needed?

Comment (turt2live) [reply tests/10apidoc/44account_data.pl]: looks like the other functions here also have it...

Comment (erikjohnston) [reply tests/10apidoc/44account_data.pl]: Github lost my comment that I noticed that :(

I have no idea what its doing with its life

**Comment by richvdh** [comment]
I guess this needs to wait for matrix-org/synapse#4303 to land?

**Comment by turt2live** [comment]
yes, and the status of the synapse PR is waiting for the MSC bot to run its course, which it did over the holidays.

**Pull request merged**